test/test_simulation_execute.jl

```diff
@@ -112,25 +115,27 @@ function test_load_simulation(file_path::String)
 
         @testset "Test reading and writing to the results folder" begin
             for name in stage_names
-                files = collect(readdir(sim_results.results_folder))
+                results_folder = joinpath(sim_results.results_folder, name)
+                files = readdir(results_folder)
                 for f in files
-                    rm("$(sim_results.results_folder)/$f")
+                    rm("$(results_folder)/$f")
                 end
-                rm(sim_results.results_folder)
+                rm(results_folder)
                 res = load_simulation_results(sim_results, name)
                 !ispath(res.results_folder) && mkdir(res.results_folder)
                 write_results(res)
-                loaded_res = load_operation_results(sim_results.results_folder)
+                loaded_res = load_results(results_folder) ##### check
```

Comment:
Is something outstanding here?

Reply:
The tests are passing. @Lilyhanig is there something here missing? or can I just delete the comment?